### ISSUE TITLE
COSMIC applications are not affected by icon theme

### ISSUE THREAD

**Issue by TheESN** [pop-os/cosmic-epoch#718]
Default COSMIC applications such as COSMIC Files, COSMIC Terminal etc do not apply icons from icon theme themselves. Icon themes have generic icons for such applications, which could be applied if this theme was changed

Examples from Yaru icon theme for applications that COSMIC currently has

File manager
![изображение](https://github.com/user-attachments/assets/9d776929-c717-4b7b-beaf-d976c49ac08c)

Terminal
![изображение](https://github.com/user-attachments/assets/d610a632-a84c-41b2-ae3d-217922a5f043)

Settings
![изображение](https://github.com/user-attachments/assets/7340f622-1fa9-49cd-b4ee-1f28f8194c14)

Software Store
![изображение](https://github.com/user-attachments/assets/d56e7d19-3250-48da-9358-91e2ca21cbc8)

Text Editor
![изображение](https://github.com/user-attachments/assets/7ff8e3fb-c934-4b57-aa6a-79e7e0b7aefe)

Screenshot App
![изображение](https://github.com/user-attachments/assets/7ed48e13-6aad-4b35-a717-da0412463fae)

**Comment by NadimDeeb** [comment]
Yup. Facing the same issue. You'll have to change them manually for now (although you'll have to through each one) or install something that isn't COSMIC.
